### PR TITLE
setup protractor configuration

### PR DESCRIPTION
**Pull request by StevenMaMei**
This is the first part of the workshop.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1 @@
+ @aperdomob @germandavid85 @luigisamurai @yesidbalvin @Scot3004 @santirogu
```
Comment (yesid-lopez) [inline .github/CODEOWNERS]: please add to Holmes, Reyes, Valery, and I as codeowners....Those ones are not needed

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+*.msm
+*.msp
+
+# Windows shortcuts
+*.lnk
+
+# Typescript
+dist
+
+# End of https://www.toptal.com/developers/gitignore/api/windows,visualstudiocode,node
```
Comment (yesid-lopez) [inline .gitignore]: add EoF... It is enough if you add a blank line

File: comandos.txt [hunk 3]
```diff
@@ -0,0 +1,3 @@
+84.0.4147.30
```
Comment (yesid-lopez) [inline comandos.txt]: this file should not be uploaded

File: package.json [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+    "protractor"
+  ],
+  "author": "Steven stevenm1999@hotmail.es StevenMaMei",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/StevenMaMei/protractor-workshop-2020/issues"
+  },
+  "homepage": "https://github.com/StevenMaMei/protractor-workshop-2020#readme",
+  "dependencies": {
+    "protractor": "^7.0.0"
```
Comment (yesid-lopez) [inline package.json]: move protractor as a dev dependency

File: .editorconfig [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+indent_style = space
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+indent_size = 2
+
+[*.md]
+indent_size = 4
+trim_trailing_whitespace = false
```
Comment (holgiosalos) [inline .editorconfig]: agrega un salto de linea al final del archivo. Puedes notar que hace falta un salto de linea en un archivo por el simbolo menos en rojo que te muestra github. 
image

**Review by holgiosalos — CHANGES_REQUESTED** [review]

File: .github/CODEOWNERS [hunk 6]
```diff
@@ -0,0 +1 @@
+@holgiosalos @renardete @leonleo997 @valeryibarra
```
Comment (yesid-lopez) [inline .github/CODEOWNERS]: Add a line break because the end of file is missing

Comment (yesid-lopez) [reply .github/CODEOWNERS]: Do not forget the `*`  at the beginning 
image

Comment (StevenMaMei) [reply .github/CODEOWNERS]: Done

**Review by yesid-lopez — APPROVED** [review]

**Comment by yesid-lopez** [comment]
@holgiosalos remove your block

**Pull request merged**